tests/test_render.py

```diff
@@ -475,6 +475,32 @@ def test_initial_state(self):
 
         assert_clean_console(self)
 
+    def test_array_of_nully_child(self):
+        app = Dash(__name__)
+        app.layout = html.Div(id='nully-wrapper', children=[0])
```

Comment:
This worked and needs to keep working

Reply:
nice. I'd call this falsy though, not nully.

Reply:
good point